source/includes/api/tables/progress-response.rst

```diff
@@ -51,13 +51,17 @@
    * - ``collectionCopy``
        ``.estimatedTotalBytes``
      - integer
-     - Estimated total number of bytes to be copied.
+     - Estimated total number of bytes globally to be copied.
 
    * - ``collectionCopy``
        ``.estimatedCopiedBytes``
      - integer
      - Estimated number of bytes which have been copied to the
-       destination cluster.
+       destination cluster by this instance of ``mongosync``.  
+       
+       To calculate the total estimated progress, add the value of  the
+       ``estimatedCopiedBytes`` field for each instance of ``mongosync`` and
```

Comment:
Consider either:

``estimatedCopiedBytes`` field for each source cluster of ``mongosync`` 
``estimatedCopiedBytes`` field for each source instance of ``mongosync``

Reply:
Believe the global/per focus here rests on a 1:N use case, where multiple mongosync instances are at work on one source, rather than multi-source sync.  Will leave this up in case the external reviewer wants to correct me.